source/sphinx_extensions/plugin_directory/templates/action.rst

```diff
@@ -32,11 +32,19 @@
      </ul>
      <div class="tab-content">
        <div id="cli" class="tab-pane fade in active">
-         <pre>
-           {{- cli_help -}}
-         </pre>
+         <h4>Docstring:</h4>
+         <pre>{{- cli_help -}}</pre>
        </div>
        <div id="api" class="tab-pane fade">
+       <h4>Import:</h4>
+
+.. command-block::
+
+       from qiime2.plugins.{{plugin_api_name}}.actions import {{action_api_name}} as {{action_api_name}}
```

Comment:
The last part of this is redundant:

```python
from qiime2.plugins.example_plugin.actions import example_action as example_action
```

is the same as just:

```python
from qiime2.plugins.example_plugin.actions import example_action
```

Reply:
Sure is, @ebolyen. My thinking was that it gave users new to Python an easy opportunity to rename their import, but maybe this is too much like "teaching users Python"?

Reply:
Unfortunately the more likely result would be that everyone would copy/paste the import with the redundant `as` expression. So I would leave it out, as others might also interpret the redundant expression as having some significance.

(Also, generally speaking, in Python people don't rename imports as a rule unless there's something to be gained from it (or its convention like `numpy` -> `np`).)

Reply:
Removed. Thanks!